src/backend/PhotoAPI/Domains/DataTransferObjects/OriginalPhotoDTO.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Domains.DataTransferObjects
+{
+	public class OriginalPhotoDTO
+	{
+		public string PhotoUrl { get; set; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this property type to 'System.Uri'.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=5141404)